test/performance/odp_l2fwd.c

```diff
@@ -41,6 +41,12 @@
 /* Maximum pktio index table size */
 #define MAX_PKTIO_INDEXES      1024
 
+/* Default vector size */
+#define DEFAULT_VEC_SIZE       4
+
+/* Default vector timeout */
+#define DEFAULT_VEC_TMO       (200 * 1000) /* 200ms */
```

Comment:
Please use `ODP_TIME_MSEC_IN_NS` or `ODP_TIME_USEC_IN_NS` here.

Reply:
Thanks for the review. Will change in v2.